packages/squiggle-lang/src/public/SqValue/SqPlot.ts

```diff
@@ -260,22 +255,11 @@ export class SqDistFnPlot extends SqAbstractPlot<"distFn"> {
 export class SqScatterPlot extends SqAbstractPlot<"scatter"> {
   tag = "scatter" as const;
 
-  private buildSampleSetDist(
-    dist: BaseDist,
-    env: Env
-  ): Result.result<SqSampleSetDistribution, SqError> {
-    const sampleSetResult = SampleSetDist.fromDist(dist, env);
-    if (!sampleSetResult.ok) {
-      return Result.Err(new SqOtherError("Conversion to SampleSet failed"));
-    }
-    return Result.Ok(new SqSampleSetDistribution(sampleSetResult.value));
-  }
-
-  xDist(env: Env): Result.result<SqSampleSetDistribution, SqError> {
-    return this.buildSampleSetDist(this._value.xDist, env);
+  xDist(): SqSampleSetDistribution {
```

Comment:
It's a nice coincidence that scatter plots require sample set dists now, otherwise I'd have to require `rng` prop.